indico/migrations/versions/20210630_0902_ec793f40439e_add_anonymous_submissions_table_for_.py

```diff
@@ -0,0 +1,39 @@
+"""Add anonymous_submissions table for surveys
+
+Revision ID: ec793f40439e
+Revises: 90384b9b3d22
+Create Date: 2021-06-30 09:02:33.328010
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = 'ec793f40439e'
+down_revision = '90384b9b3d22'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        'anonymous_submissions',
+        sa.Column('survey_id', sa.Integer(), nullable=False),
+        sa.Column('user_id', sa.Integer(), nullable=False),
+        sa.ForeignKeyConstraint(['survey_id'], ['event_surveys.surveys.id'],
+                                name=op.f('fk_anonymous_submissions_survey_id_surveys')),
```

Comment:
all the names in the constraints are autogenerated by SA so they can be removed here